src/header.wppl

```diff
@@ -469,11 +469,49 @@ var OptimizeThenSample = function(wpplFn, options) {
   return SampleGuide(wpplFn, opts);
 };
 
-var Infer = function(options, maybeFn) {
-  if (!util.isObject(options)) {
-    util.fatal('Infer: expected first argument to be an options object.');
+/*
+ * DefaultInfer() called when no options are specified
```

Comment:
Use `maxEnumTreeSize` or something similarly explicit instead of `enumTol`? Also, use `estimatedTreeSize`/`maxTreeSize` in the enumeration file instead of "complexity"?